src/main/java/me/totalfreedom/totalfreedommod/blocking/EventBlocker.java

```diff
@@ -75,7 +78,8 @@ public void onEntityExplode(EntityExplodeEvent event)
             return;
         }
 
-        event.setYield(0.0F);
```

Comment:
I think we should still allow it to blow up other tnt blocks, or else it isn't very fun.

Reply:
Then make server massive lag?...

Reply:
True....

Reply:
But we could make a command like /explosion block add/remove/purge <block> to let any explosive TNT to break that specified block, like

If i want to break bedrock with TNT

/explosives block add bedrock , then TNT would be able to break that block only!

Oblivious we could add a blacklist.

Reply:
sounds kinda really bad, like do you seriously want to add something useless like that? if it's an explosive, it should be destroying everything nearby, not just one single block type.

also, setting the yield doesn't really cause server lag.

Reply:
@robingall2910 
The yeld is a differenty type, i kept the yeld on vanilla, but i made this change

+        event.blockList().clear();
 

This clears any block list added in the explosion , and this makes it doesn't break any block.